aead/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "aead"
+version = "0.1.0"
+authors = ["James Cape <james@mobilecoin.com>"]
+description = """
+The RFC5116 AEAD API, as a trait.
+"""
+
+[features]
+std = []
+dev = []
```

Comment:
What is this supposed to be for?

Reply:
In other crates `dev` feature is used for functions and macros which are used for tests and benchmarks. Since right now we don't have a `dev` module in this crate this feature can be removed.

Reply:
build_std.sh requires both an std and dev feature, if that's not required I'd just as soon remove both.

Reply:
Can you replace `--features std,dev` with `--all-features` in `build_std.sh` as part of this PR? After that you can remove those features.